src/components/NavBar/SearchBar.tsx

```diff
@@ -110,19 +113,40 @@ export const SearchBarDropdown = ({ toggleOpen, tokens, collections, hasInput }:
     )
 
   const collectionSearchResults =
-    collections.length > 0 ? (
-      <SearchBarDropdownSection
-        hoveredIndex={hoveredIndex}
-        startingIndex={isNFTPage ? 0 : tokens.length}
-        setHoveredIndex={setHoveredIndex}
-        toggleOpen={toggleOpen}
-        suggestions={collections}
-        header={'NFT Collections'}
-      />
+    phase1Flag === Phase1Variant.Enabled ? (
+      collections.length > 0 ? (
+        <SearchBarDropdownSection
+          hoveredIndex={hoveredIndex}
+          startingIndex={isNFTPage ? 0 : tokens.length}
+          setHoveredIndex={setHoveredIndex}
+          toggleOpen={toggleOpen}
+          suggestions={collections}
+          header={'NFT Collections'}
+        />
+      ) : (
+        <Box className={styles.notFoundContainer}>No NFT collections found.</Box>
```

Comment:
translate these rendered strings

Reply:
localization is not in the scope of this work and is to be addressed in a follow up PR